src/Actions/contacts.js

```diff
@@ -0,0 +1,15 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import 'react-native-get-random-values';
+import { v4 as uuidv4 } from 'uuid';
+const id = uuidv4();
```

Comment:
this will work only one time. so all new IDs will the same.
move Id creation inside action